package.json

```diff
@@ -45,5 +45,13 @@
     "router",
     "route-recognizer",
     "rsvp"
-  ]
+  ],
+  "dependencies": {
```

Comment:
The point of `peerDependencies` is that we expect the "host" (the thing that depends upon `route-recognizer`) to provide these dependencies so that it can choose its own version.

We specify `devDependencies` so that our own test suite functions (since in that context it is the "host").